_data/software/phpmailer.yml

```diff
@@ -0,0 +1,33 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: |
+      A full-featured email creation and transfer class for PHP.
+    fr: Une classe complète de création et de transfert d'e-mails pour PHP.
+category: '43232400'
+homepageURL:
+  en: 'https://github.com/Synchro/PHPMailer'
```

Comment:
Looks like it has a bunch of stars, but shouldn't it point to the original instead of the fork? https://github.com/PHPMailer/PHPMailer